### PR TITLE
apigee-x-trial-provision: minor changes to readme and echo'd statements

### PR DESCRIPTION
**Pull request by OmidTahouri**
What's changed, or what was fixed?

- devrel/tools/apigee-x-trial-provision

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Comment by laughingbiscuit** [comment]
@yuriylesyuk please review

**Comment by laughingbiscuit** [comment]
lgtm - just pending CODEOWNER confirmation

File: tools/apigee-x-trial-provision/README.md [hunk 1]
```diff
@@ -4,16 +4,15 @@
 
 ## Description
 
-This script creates an Apigee X trial organization and instance. It uses
-gcloud command to create a hybrid runtime and adds an envoy proxy and
-Google Cloud Load Balancer,
-[GCLB](https://cloud.google.com/load-balancing/docs) for external exposure.
+This script creates an Apigee X evaluation organization and instance. It uses
+`gcloud` commands to create an Apigee runtime instance,
+a proxy [MIG](https://cloud.google.com/compute/docs/instance-groups) and a [GCLB](https://cloud.google.com/load-balancing/docs) for external exposure.
```
Comment (danistrebel) [inline tools/apigee-x-trial-provision/README.md:9]: Line length -> see linter error

**Pull request merged**